cmd/main.go

```diff
@@ -302,18 +306,35 @@ func main() {
 		crdUtils = controllers.NewDummyCRDUtils()
 	}
 
+	gc := garbagecollector.NewGarbageCollectors(db, log.WithField("pkg", "garbage_collector"), hostApi, clusterApi, objectHandler, lead)
+
+	if Options.EnableDeregisterInactiveGC {
+
+		deregisterWorker := thread.New(
+			log.WithField("garbagecollector", "Deregister Worker"),
+			"Deregister Worker",
```

Comment:
worker?

Reply:
no? an entity that works and deregisters